src/Common/src/CoreLib/System/Collections/ObjectModel/Collection.cs

```diff
@@ -69,6 +69,8 @@ public void Add(T item)
             InsertItem(index, item);
         }
 
+        public void AddRange(IEnumerable<T> collection) => InsertItemsRange(items.Count > 0 ? items.Count : 0, collection);
```

Comment:
Changes under CoreLib don't need to be made here. The changes from coreclr will get auto-mirrored. For example see https://github.com/dotnet/corefx/pull/35731

Reply:
Understood, didn't realize exactly how the process works since this is my first CoreLib & .NET Core PR. I will remove the changes here from the PR and make changes to CoreLib in #23018

Reply:
No problem. We should update the [docs](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/changing-corelib.md) to include the mirroring aspect of it to make it easier for new contributors to make changes between the two repos.

cc @Anipik

Reply:
> We should update the [docs](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/changing-corelib.md) to include the mirroring aspect of it to make it easier for new contributors to make changes between the two repos.

My feedback on this item is making the docs more accessible throughout the project or in the contributing guide. I had to jump through a few different docs to find this one which is why I didn't realize it was needed originally.

Perhaps it would be best to include in the [CoreLib Readme](https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/README.md). Now that I look back at that README it does mention mirroring but doesn't reference the doc so it wasn't clear.

Reply:
cc: @karelz